build.cake

```diff
@@ -1,29 +1,43 @@
 //////////////////////////////////////////////////////////////////////
 // TOOLS
 //////////////////////////////////////////////////////////////////////
-#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+#module nuget:?package=Cake.DotNetTool.Module&version=0.1.0
```

Comment:
Wondering why this seems to have gone backwards in version?

Reply:
There was an API change for `GetFiles(...)` method in the newer version of cake which expects a glob pattern instead of what it was previously, and the docs were a little difficult to navigate. I've changed this prior to "make it work", but I totally forgot about addressing this afterwards.

I'm going to close this PR in favour of this newer PR #28 (to make sure our pipeline is happy without me fiddling around) which uses Cake version `0.4.0` with the glob pattern matching